omero/developers/Web/PublicData.txt

```diff
@@ -80,3 +83,24 @@ Here is how to go about setting this up on your OMERO.web install:
     can always access the login page using the following link
     `\http://your_host/webclient/login/`.
 
+
+Reusing OMERO session
+---------------------
+
+Since OMERO 4.4, the OMERO.web framework has supported password-less, OMERO
+session key-based authentication. For example a direct link to image will look
+as follows:
+
+::
+
+    https://your_host/webgateway/img_detail/IMAGE_ID/?server=SERVER_ID&bsession=OMERO_SESSION_KEY
+
+.. note::
+
+    SERVER_ID should match the index from the list set using
+    :property:`omero.web.server_list` from the server session
```

Comment:
You need to remove the `:property:` link here - these don't exist on this branch.